### PR TITLE
doc(release): add release note for centreon-connector-ssh-21.04.4

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [x] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 1]
```diff
@@ -680,6 +680,20 @@ New release.
...
+#### Improvements
+
+- Fixed an issue that could crash the SSH connector
+
+#### Bug fixes
+
+- UNKNOWN status is displayed instead of WARNING and CRITICAL when using the perl connector
+- Fixed a memory leak issue in the Perl connector
+
+
```
Comment (omercier) [inline versioned_docs/version-21.04/releases/centreon-core.md:696]:
```suggestion
#### Bug fixes

- Fixed an issue that could crash the SSH connector
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 2]
```diff
@@ -665,6 +665,13 @@ Nouvelle version majeure.
 
 ## Centreon Connector SSH
 
+### 21.04.4
+
+Release date: `null`
+
+#### Bug fixes
+
+- Fixed an issue that could crash the SSH connector
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:674]:
```suggestion
- Fixed an issue that could cause the SSH connector to crash

```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 3]
```diff
@@ -680,6 +680,13 @@ New release.
 
 ## Centreon Connector SSH
 
+### 21.04.4
+
+Release date: `null`
+
+#### Bug fixes
+
+- Fixed an issue that could crash the SSH connector
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:689]:
```suggestion
- Fixed an issue that could cause the SSH connector to crash

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by chgautier — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 4]
```diff
@@ -665,6 +665,14 @@ Nouvelle version majeure.
 
 ## Centreon Connector SSH
 
+### 21.04.4
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:668]: ### 21.04.4

Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:668]:
```suggestion
### 21.04.4
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 5]
```diff
@@ -680,6 +680,14 @@ New release.
 
 ## Centreon Connector SSH
 
+### 21.04.4
```
Comment (chgautier) [inline versioned_docs/version-21.04/releases/centreon-core.md:683]:
```suggestion
### 21.04.4
```

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 6]
```diff
@@ -665,6 +665,14 @@ Nouvelle version majeure.
 
 ## Centreon Connector SSH
 
+### 21.04.4
+
+Release date: `null`
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:670]:
```suggestion
Release date: `June 10, 2022`
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 7]
```diff
@@ -680,6 +680,14 @@ New release.
 
 ## Centreon Connector SSH
 
+### 21.04.4
+
+Release date: `null`
```
Comment (chgautier) [inline versioned_docs/version-21.04/releases/centreon-core.md:685]:
```suggestion
Release date: `June 10, 2022`
```

**Review by chgautier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**